apps/coscrad-frontend/src/components/resources/digital-text/digital-text-detail.full-view.presenter.tsx

```diff
@@ -40,7 +40,13 @@ export const DigitalTextDetailFullViewPresenter = ({
     };
 
     return (
-        <ResourceDetailFullViewPresenter name={name} id={id} type={ResourceType.digitalText}>
+        <ResourceDetailFullViewPresenter
+            name={name}
+            id={id}
+            type={ResourceType.digitalText}
+            // TODO flow through the contributions
```

Comment:
Add a story link here so this doesn't get lost.

Reply:
[Here](https://github.com/COSCRAD/coscrad/pull/591#discussion_r1614097555) is the story.